hubblestack/config/__init__.py

```diff
@@ -0,0 +1,1877 @@
+# -*- coding: utf-8 -*-
+'''
+All salt configuration loading and defaults should be in this module
+'''
+
+# Import python libs
+from __future__ import absolute_import, print_function, unicode_literals, generators
+import os
+import re
+import sys
+import glob
+import time
+import codecs
+import logging
+import types
+from copy import deepcopy
+
+# pylint: disable=import-error,no-name-in-module
+from salt.ext import six
```

Comment:
No Six module.
All salt.xyz dependencies must be removed below.

And looks like we have copied code which we don't need. Example zeromq.

Reply:
Removed config as it will be ported by Paul.